apps/dashboard/app/javascript/system_status.js

```diff
@@ -0,0 +1,17 @@
+import { bcPollDelay } from './config'
+import { replaceHTML } from './turbo_shim'
+
+function poll() {
+  url = "";
+  fetch(url, { headers: { Accept: "text/vnd.turbo-stream.html" } })
```

Comment:
I think this URL set to "" is a bit dangerous as this may be used in other places. I'd suggest adding the route to the `app/views/layouts/_config.html.erb` and then adding the corresponding method to the `config.js` file to extract the correct URL.